extensions/amp-connatix-player/0.1/amp-connatix-player.js

```diff
@@ -47,14 +65,24 @@ export class AmpConnatixPlayer extends AMP.BaseElement {
    * (play/pause etc)
    * @private
    * @param {string} command
+   * @param {Object=} opt_args
    */
-  sendCommand_(command) {
-    if (this.iframe_ && this.iframe_.contentWindow) {
-      // Send message to the player
-      this.iframe_.contentWindow./*OK*/ postMessage(
-        command,
-        this.iframeDomain_
-      );
+  sendCommand_(command, opt_args) {
+    if (this.playerReadyPromise_) {
```

Comment:
nit: reverse condition and early return instead

Reply:
updated